svd2py/objects.py

```diff
@@ -29,6 +29,7 @@ def _get_bool(self, value: str) -> bool:
             raise ValueError(f"Invalid boolean value {value}")
 
     def _get_int(self, value: str) -> int:
+        value = value.strip("#")
```

Comment:
Thanks for pointing that out. The `#` according to SVD spec means that the number is in binary format. Your solution is not fully accurate. The `value` has to be cast properly to int. I would see something like that:
```suggestion
        if value.startswith("#"):
            return int(value.strip("#"), 2)
```

Reply:
Didn't notice that, thanks.